administrator-manual/en/firewall.rst

```diff
@@ -165,22 +188,25 @@ Allow guest's network to access all the services listening on Server1:
 * Destination: Server1 
 * Service: -
 
-.. _multi-wan-section:
+.. _wan-section:
 
-Multi WAN
-=========
+WAN
+===
 
 The term :dfn:`WAN` (Wide Area Network) refers to a public network outside the server, usually connected to the Internet. 
 A :dfn:`provider` is the company that actually manages the :index:`WAN` link.
 
-The system supports up to 15 WAN connections. 
-If the server has two or more configured red interfaces, it is required to correctly fill :guilabel:`Link weight`, 
-:guilabel:`Inbound bandwidth` and :guilabel:`Outbound bandwidth` fields from the :guilabel:`Network` page. 
+All WAN network interfaces are labeled with the red role and are listed on the top of the page, just below bandwidth usage charts.
+Rules can be created under the :guilabel:`Rules` section on the same page.
+
+If the server has two or more configured red interfaces, it is required to correctly fill, 
+:guilabel:`Download bandwidth` and :guilabel:`Upload bandwidth` fields from the :guilabel:`Network` page.
+Download and upload bandwidth can be automatically calculated using :guilabel:`Speedtest` button.
 
 Each provider represents a WAN connection and is associated with a network adapter. 
-Each provider defines a :dfn:`weight`: higher the :index:`weight`, higher the priority of the network card associated with the provider. 
+Each provider defines a :dfn:`weight`: to an higher the :index:`weight` correspond an higher priority of the network card associated with the provider. 
```

Comment:
```suggestion
Each provider defines a :dfn:`weight`: the higher the weight, the higher the priority of the network card associated with the provider. 
```

Reply:
I do not like the old sentence.

Reply:
There's a mistake in the new sentence too: 

```diff
- to an higher the :index:`weight`
+ to an higher :index:`weight`
```